linters/index.js

```diff
@@ -3,8 +3,10 @@
 
 const Bandit = require('./bandit')
 const Gosec = require('./gosec')
+const TSLint = require('./tslint')
 
 module.exports = {
   BANDIT: new Bandit(),
-  GOSEC: new Gosec()
+  GOSEC: new Gosec(),
+  TSLint: new TSLint()
```

Comment:
Is there an implied convention here that all exports are upper case? We don't seem to have an explicit coding style in [CONTRIBUTING.md](https://github.com/vmware/precaution/blob/026b54e77efe9d8f2a50356fdc6ee37d11b93722/CONTRIBUTING.md#code-style) yet, but this would match [npm-coding-style](https://docs.npmjs.com/misc/coding-style.html#case-naming-etc), at least.

Reply:
We are using JavaScipt style guidelines called - JavaScript Standart.
Maybe won't be a bad idea to change all exports with upper cases.

Reply:
Either way, we should be consistent.